### PR TITLE
doc: update Ubuntu GSG for ACRN V2.0.

### PR DESCRIPTION
**Pull request by shenfang2019**
Signed-off-by: fangfang.shen <fangfang.shen@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by dbkinder** [comment]
I'm going to merge this and submit a PR to fix the doc build error.

File: doc/getting-started/rt_industry_ubuntu.rst [hunk 1]
```diff
@@ -350,20 +352,18 @@ Follow these instructions to build the RT kernel.
 #. Build the kernel:
 
    .. code-block:: none
-
+      $ cp x86-64_defconfig .config
```
Comment (dbkinder) [inline doc/getting-started/rt_industry_ubuntu.rst:355]: must be a blank line before the body of the code block

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/getting-started/rt_industry_ubuntu.rst [hunk 2]
```diff
@@ -351,19 +353,18 @@ Follow these instructions to build the RT kernel.
...
 
    .. code-block:: none
 
-      $ sudo mount /dev/sda1 /mnt
-      $ sudo cp arch/x86/boot/bzImage /mnt/EFI/
-      $ sudo umount /mnt
       $ sudo mount /dev/sda2 /mnt
-      $ sudo cp linux-4.19.72-rt25-x86.tar.gz -P /mnt/usr/lib/modules/ && cd /mnt/usr/lib/modules/
-      $ sudo tar zxvf linux-4.19.72-rt25-x86.tar.gz
+      $ sudo cp arch/x86/boot/bzImage /mnt/boot/
```
Comment (gvancuts) [inline doc/getting-started/rt_industry_ubuntu.rst:365]: This seems very suspicious... is the content of the `/boot` folder really on `/dev/sda2`??

File: doc/getting-started/rt_industry_ubuntu.rst [hunk 3]
```diff
@@ -351,19 +353,18 @@ Follow these instructions to build the RT kernel.
...
+      $ sudo tar -zxvf linux-4.19.72-rt25-x86.tar.gz -C /mnt/lib/modules/
+      $ sudo cp -r /mnt/lib/modules/lib/modules/4.19.72-rt25 /mnt/lib/modules/
```
Comment (gvancuts) [inline doc/getting-started/rt_industry_ubuntu.rst:367]: Why??

File: doc/getting-started/rt_industry_ubuntu.rst [hunk 4]
```diff
@@ -49,10 +49,12 @@ Install Ubuntu on the SATA disk
 -------------------------------
 
 .. note:: The WHL Maxtang machine contains both an NVMe and SATA disk.
+   Before you install the Ubuntu User VM on the SATA disk, either
+   remove the NVME disk or delete it's blocks.
```
[the remark this answers is not in the document]
Comment (shenfang2019) [reply doc/getting-started/rt_industry_ubuntu.rst:53]: while entering "installation type“ during installation procedure, select /dev/nvme0n1p1, /dev/nvme0n1p2, /dev/nvme0n1p3, then select"-" to delete nvme blocks.

File: doc/getting-started/rt_industry_ubuntu.rst (hunk 2 above)
Comment (shenfang2019) [reply doc/getting-started/rt_industry_ubuntu.rst:365]: sda     8:0    0 119.2G  0 disk
├─sda1 8:1    0   762M  0 part /boot/efi
├─sda2 8:2    0 111.8G  0 part /
└─sda3 8:3    0   6.8G  0 part [SWAP]
only /boot/efi is in /dev/sda1, others of /boot all in /dev/sda2.

File: doc/getting-started/rt_industry_ubuntu.rst (hunk 3 above)
Comment (shenfang2019) [reply doc/getting-started/rt_industry_ubuntu.rst:367]: linux-4.19.72-rt25-x86.tar.gz includes some folders, like lib/modules/4.19.72-rt25 and boot.

File: doc/getting-started/rt_industry_ubuntu.rst (hunk 2 above)
Comment (gvancuts) [reply doc/getting-started/rt_industry_ubuntu.rst:365]: Ah yes, I installed Ubuntu over the week-end and now I see this. Thanks!

File: doc/getting-started/rt_industry_ubuntu.rst (hunk 3 above)
Comment (gvancuts) [reply doc/getting-started/rt_industry_ubuntu.rst:367]: Could we do this like this then?
```
$ sudo tar -zxvf linux-4.19.72-rt25-x86.tar.gz -C /mnt/
```

File: doc/getting-started/rt_industry_ubuntu.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (gvancuts) [reply doc/getting-started/rt_industry_ubuntu.rst:53]: @dbkinder can you add a note in that sense?

File: doc/getting-started/rt_industry_ubuntu.rst (hunk 3 above)
Comment (shenfang2019) [reply doc/getting-started/rt_industry_ubuntu.rst:367]: yes, I think we can decompress the "tar"packet anywhere, only need copy the related folder into the right folder.

Comment (shenfang2019) [reply doc/getting-started/rt_industry_ubuntu.rst:367]: Ah, I got your point, that's more easy and directly.

**Pull request merged**